Dockerfile

```diff
@@ -35,6 +35,10 @@ RUN ["chmod", "-R", "u=rwX,go=rX", "/rootfs"]
 
 FROM scratch
 
+LABEL org.opencontainers.image.documentation='https://icinga.com/docs/icinga-db/latest/doc/01-About/' \
+      org.opencontainers.image.source='https://github.com/Icinga/icingadb' \
+      org.opencontainers.image.licenses='GPL-2.0+'
```

Comment:
According to the docs, `.licenses` should be an SPDX License Expression.

```suggestion
      org.opencontainers.image.licenses='GPL-2.0-or-later'
```

Reply:
Nope.

```suggestion
      org.opencontainers.image.licenses='GPL-2.0-only'
```

Reply:
Are you sure? The Dockerfile has `GPLv2+` in its header.

https://github.com/Icinga/docker-icingadb/blob/a362254223b65f9c74e96395710b6d346c63c368/Dockerfile#L1

Reply:
The [LICENSE file](https://github.com/Icinga/icingadb/blob/main/LICENSE#L299) states `either version 2 of the License, or (at your option) any later version.`. In my opinion that would be the final authorative legal source and indicates GPL-2.0+/GPL-2.0-or-later

Reply:
We can also drop the label for now. It is not super important to tag that to the image (at least for us). If you want I can remove it.